staging/src/k8s.io/client-go/discovery/fake/discovery_test.go

```diff
@@ -17,11 +17,14 @@ limitations under the License.
 package fake_test
 
 import (
+	"errors"
+	"k8s.io/apimachinery/pkg/runtime"
```

Comment:
Move the k8s import down into the block of hte other k8s imports.

Reply:
ok, [done](https://github.com/kubernetes/kubernetes/pull/114291/commits/67fe0ed57b6d8d461dd8b9bf8eb3a0b719c21a62)